### PR TITLE
refactor(lld): nfts gallery medias in newArch

### PR DESCRIPTION
**Pull request by LucasWerey**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - None

### 📝 Description

Refactor of nfts gallery medias inside newArch to prepare the integration of ordinals and improve readability 
Change from class components to RFC. 
Creation of the folder Collectibles where all collectibles will be centralised
Creation of a FF to hide the implementation of newArch
For the moment none of the components added in NewArch are in use 
 

### ❓ Context

- **JIRA or GitHub link**: [LIVE-12454](https://ledgerhq.atlassian.net/browse/LIVE-12454) &  [LIVE-12457](https://ledgerhq.atlassian.net/browse/LIVE-12457)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-12454]: https://ledgerhq.atlassian.net/browse/LIVE-12454?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by KVNLS** [comment]
Could you use a Feature Flag to make this migration, please? As we did for the Market page on LLM we need to be able to roll back to the previous architecture without the need to do a new release. You can use the Router to display the correct page whether the FF is activated or not. Plus we need to avoid references to the newArch in the old one.
You can duplicate components if needed and we will remove them once we know the migration is working well in a second time.
Thanks in advance!

File: apps/ledger-live-desktop/src/newArch/Collectibles/components/CollectionName.tsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
-import React, { memo, useMemo } from "react";
+import React, { memo } from "react";
 import { useNftCollectionMetadata } from "@ledgerhq/live-nft-react";
 import { Account, ProtoNFT } from "@ledgerhq/types-live";
 import NFTCollectionContextMenu from "~/renderer/components/ContextMenu/NFTCollectionContextMenu";
-import Skeleton from "~/renderer/components/Nft/Skeleton";
+import { Skeleton } from "LLD/Collectibles/components";
```
Comment (lvndry) [inline apps/ledger-live-desktop/src/newArch/Collectibles/components/CollectionName.tsx:5]: what's the difference between `LLD` and `~` ?

File: apps/ledger-live-desktop/src/newArch/Collectibles/components/index.tsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export { default as Skeleton } from "./Skeleton";
```
Comment (lvndry) [inline apps/ledger-live-desktop/src/newArch/Collectibles/components/index.tsx:1]: Why not just use named exports in the files themsevles ?
And do `export { Skeleton } from "./Skeleton"`

File: apps/ledger-live-desktop/src/newArch/Collectibles/components/CollectionName.tsx (hunk 1 above)
Comment (LucasWerey) [reply apps/ledger-live-desktop/src/newArch/Collectibles/components/CollectionName.tsx:5]: LLD is the alias for ~/newArch/

**Review by gre — APPROVED** [review]

**Pull request merged**